package.json

```diff
@@ -9,9 +9,11 @@
     "dev": "nodemon bot.js"
   },
   "keywords": [],
-  "author": "",
+  "author": "Pranavius & Tuskyouup",
```

Comment:
Thanks for the credit but that's okay. I've not contributed any code here. This is all you buddy.

Reply:
1. If it were just me, the outputs would look awful and disorganized haha
2. I have a feeling that as we move forward you'll make some code contributions as well. I'm speaking it into existence (well, writing in this case).

Reply:
lol ok